src/notify.js

```diff
@@ -71,6 +83,11 @@ Notify.isSupported = function(perm) {
         throw new Error('You must only call this before calling Notification.requestPermission(), otherwise this feature detect would trigger an actual notification!');
     }
 
+    if (Notify.serviceWorkerSupported()) {
```

Comment:
That's not the point - even if we do use a major version increment (which we will), it would still remain broken for Chrome on Android. Not everyone is using this lib with a SW yet.